core/embed/projects/bootloader/main.c

```diff
@@ -64,27 +57,16 @@
 #include <sec/hash_processor.h>
 #endif
 
-#include <io/usb.h>
-#include "version.h"
-
+#include "antiglitch.h"
 #include "bootui.h"
-#include "messages.h"
-#include "rust_ui.h"
 #include "version_check.h"
+#include "workflow/workflow.h"
 
 #ifdef TREZOR_EMULATOR
 #include "SDL.h"
 #include "emulator.h"
 #endif
 
-#define USB_IFACE_NUM 0
-
-typedef enum {
-  SHUTDOWN = 0,
-  CONTINUE_TO_FIRMWARE = 0xAABBCCDD,
-  RETURN_TO_MENU = 0x55667788,
-} usb_result_t;
-
 void failed_jump_to_firmware(void);
 
 CONFIDENTIAL volatile secbool dont_optimize_out_true = sectrue;
```

Comment:
slightly unsure why `allow_jump_1` was moved to antiglitch but `jump_to_firmware` stayed here...?

Reply:
`allow_jump_1` needed to moved as it is accessed from workflows, where as `jump_to_firmware` is only accessed here.